### ISSUE TITLE
java.lang.NoSuchMethodError: com.tbruyelle.rxpermissions2.RxPermissions.getInstance

### ISSUE THREAD

**Issue by 452MJ** [miguelbcr/RxPaparazzo#54]
I found RxPaparazzo2 is still use RxPermission with an old version to get RxPermission instance.

```
package com.miguelbcr.ui.rx_paparazzo2.interactors.GrantPermissions

RxPermissions.getInstance(targetUi.activity())
        .requestEach(permissions)
        .buffer(permissions.length)
        ...
        .toObservable();
```

**Comment by miguelbcr** [comment]
Hi @452MJ ,
this is because RxPaparazzo is using RxPermissions 0.9.x and rxPaparazzo2 is using RxPermissions 0.8.x , where constructors are differents, and maybe it came into conflict with your RxPermissions version (if you are using the newest one)

Anyway I've released [0.4.3](https://github.com/miguelbcr/RxPaparazzo/releases/tag/0.4.3) and [0.4.3-2.x](https://github.com/miguelbcr/RxPaparazzo/releases/tag/0.4.3-2.x)

Try it out and let me know if crash has been solved.

**Comment by 452MJ** [comment]
@miguelbcr  (^o^)YES!The project runs perfectly, thanks for your work!
